data/azure/instance_types.json

```diff
@@ -335,111 +335,111 @@
-  "Standard_E96-48ads_v5": {"up": null, "down": "Standard_E96-24ads_v5", "superseded": {"regular": "Standard_E96ads_v5"}},
-  "Standard_E4-2as_v5": {"up": null, "down": null, "superseded": {"regular": "Standard_E4as_v5"}},
-  "Standard_E8-2as_v5": {"up": "Standard_E8-4as_v5", "down": null, "superseded": {"regular": "Standard_E8as_v5"}},
-  "Standard_E8-4as_v5": {"up": null, "down": "Standard_E8-2as_v5", "superseded": {"regular": "Standard_E8as_v5"}},
-  "Standard_E16-4as_v5": {"up": "Standard_E16-8as_v5", "down": null, "superseded": {"regular": "Standard_E16as_v5"}},
-  "Standard_E16-8as_v5": {"up": null, "down": "Standard_E16-4as_v5", "superseded": {"regular": "Standard_E16as_v5"}},
-  "Standard_E32-8as_v5": {"up": "Standard_E32-16as_v5", "down": null, "superseded": {"regular": "Standard_E32as_v5"}},
-  "Standard_E32-16as_v5": {"up": null, "down": "Standard_E32-8as_v5", "superseded": {"regular": "Standard_E32as_v5"}},
-  "Standard_E64-16as_v5": {"up": "Standard_E64-32as_v5", "down": null, "superseded": {"regular": "Standard_E64as_v5"}},
-  "Standard_E64-32as_v5": {"up": null, "down": "Standard_E64-16as_v5", "superseded": {"regular": "Standard_E64as_v5"}},
-  "Standard_E96-24as_v5": {"up": "Standard_E96-48as_v5", "down": null, "superseded": {"regular": "Standard_E96as_v5"}},
-  "Standard_E96-48as_v5": {"up": null, "down": "Standard_E96-24as_v5", "superseded": {"regular": "Standard_E96as_v5"}},
-  "Standard_GS4-4": {"up": "Standard_GS4-8", "down": null, "superseded": {"regular": "Standard_GS4"}},
-  "Standard_GS4-8": {"up": null, "down": "Standard_GS4-4", "superseded": {"regular": "Standard_GS4"}},
-  "Standard_GS5-8": {"up": "Standard_GS5-16", "down": null, "superseded": {"regular": "Standard_GS5"}},
-  "Standard_GS5-16": {"up": null, "down": "Standard_GS5-8", "superseded": {"regular": "Standard_GS5"}},
-  "Standard_DS11-1_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_DS11_v2"}},
-  "Standard_DS12-1_v2": {"up": "Standard_DS12-2_v2", "down": null, "superseded": {"regular": "Standard_DS12_v2"}},
-  "Standard_DS12-2_v2": {"up": null, "down": "Standard_DS12-1_v2", "superseded": {"regular": "Standard_DS12_v2"}},
-  "Standard_DS13-2_v2": {"up": "Standard_DS13-4_v2", "down": null, "superseded": {"regular": "Standard_DS13_v2"}},
-  "Standard_DS13-4_v2": {"up": null, "down": "Standard_DS13-2_v2", "superseded": {"regular": "Standard_DS13_v2"}},
-  "Standard_DS14-4_v2": {"up": "Standard_DS14-8_v2", "down": null, "superseded": {"regular": "Standard_DS14_v2"}},
-  "Standard_DS14-8_v2": {"up": null, "down": "Standard_DS14-4_v2", "superseded": {"regular": "Standard_DS14_v2"}},
-  "Standard_M416-208s_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_M416s_v2"}},
-  "Standard_M416-208ms_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_M416ms_v2"}},
+  "Standard_M8-2ms": {"up": "Standard_M8-4ms", "down": null, "superseded": null},
```

Comment:
I am not familiar enough with Azure Instance types to know this off-hand.  Based on "removed superseded recommendations for constrained vCPU sizes" comment in PR description, seems like this _may_ be correct

Reply:
@bryankaraffa Do you think that this PR could be approved? Or Is there anyone who could help us to check if this is right?

Reply:
Agree, think this looks good based on https://learn.microsoft.com/en-us/azure/virtual-machines/constrained-vcpu